app/views/relationships/index.html.haml

```diff
@@ -0,0 +1,22 @@
+%section.people
+  %header
+    %h2 People I Follow
+  %table.table
+    %thead
+      %tr
+        %th(width="30%")
+        %th(width="20%") Videos in Queue
+        %th(width="20%") Followers
+        %th(width="30%") Unfollow
+    %tbody
+      - @following_relationships.each do |relationship|
```

Comment:
Keep in mind `find_each` is a method on Rails collections, it won't work on a plain Ruby array.